### PR TITLE
UCT/SM/CMA: update expected memory bandwidth

### PR DESCRIPTION
**Pull request by alex-mikheev**
@yosefe @alinask please take a look

### CODE REVIEW

File: src/uct/sm/cma/cma_iface.c [hunk 1]
```diff
@@ -61,7 +61,7 @@ static ucs_status_t uct_cma_iface_query(uct_iface_h tl_iface,
                                           UCT_IFACE_FLAG_CONNECT_TO_IFACE;
     iface_attr->latency.overhead        = 80e-9; /* 80 ns */
     iface_attr->latency.growth          = 0;
-    iface_attr->bandwidth               = 10240 * 1024.0 * 1024.0; /* 10240 MB*/
+    iface_attr->bandwidth               = 11800 * 1024.0 * 1024.0;
```
Comment (yosefe) [inline src/uct/sm/cma/cma_iface.c]: shall we update knem as well?

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/4557/ for details (Mellanox internal link).

File: src/uct/sm/cma/cma_iface.c (hunk 1 above)
Comment (shamisp) [reply src/uct/sm/cma/cma_iface.c]: xpmem ?

Comment (alex-mikheev) [reply src/uct/sm/cma/cma_iface.c]: it does not support zcopy put/get

**Comment by shamisp** [comment]
Xpmem ?

**Comment by alex-mikheev** [comment]
@shamisp xpmem does not support zcopy get/put because it is a part of mm

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/4584/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/4590/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/4593/ for details (Mellanox internal link).

**Comment by shamisp** [comment]
if we don't have zcopy with xpmem it is a bug. It performs better than CMA and knem.

**Pull request merged**